src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
How can this animal stop hiding? 🙂 Better to change `hidden` value to opposite here

Reply:
I was wrong, ignore this comment 🙃